### ISSUE TITLE
Insufficient color contrast of keyboard focus indicator on "Import file", "Import GitHub repo", "Import Url" button in Import dialog is (1.248:1): A11y_Microsoft MakeCode_Import_Non-tect contrast.

### ISSUE THREAD

**Issue by kupatkar99** [microsoft/pxt-microbit#6838]
"Leveraging SWE Agent can help accelerate this accessibility bug fix; review and validate all AI generated changes before merge."

"[Check out Accessibility Insights](https://nam06.safelinks.protection.outlook.com/?url=https://accessibilityinsights.io/&data=02%7c01%7cv-stfe%40microsoft.com%7cb67b2c4b646d4f9561a208d6f4b5c39b%7c72f988bf86f141af91ab2d7cd011db47%7c1%7c0%7c636965458850501301&sdata=mxhokIKNMb22llsjXHLgU3XZibj1Qfx37rpY4PU2sfE%3D&reserved=0)! - Identify accessibility bugs before check-in and make bug fixing faster and easier."

"Contact [CAI Accessibility Support Channel](https://teams.microsoft.com/l/channel/19:DeCtYeREyQHw5jYsz9PPc3hpVw48m-tOlTkQ1jEn0eM1%40thread.tacv2/CAI%20Accessibility%20Support%20Channel?groupId=f5eb428c-9e6e-4c16-8143-33fe3961a9ae&tenantId=72f988bf-86f1-41af-91ab-2d7cd011db47) for Accessibility queries on design, development, test, usability or any processes"

**Please do not close this bug. This bug should only be closed by Trusted Tester after verification.**
### GitHub Tags:
#A11yTCS; #A11yMAS; #Win11; #DesktopWeb; #Microsoft MakeCode; #E2E_MicrosoftMakeCode_Web_May2026; #FTP; #ChromiumEdge; #Non-text Contrast; #A11ySev2; #WCAG1.4.11; #AILimited; #copilot:agent=a11yagent;

### Environment Details:
Application Name: Microsoft MakeCode
URL: https://makecode.microbit.org/beta
windows Edition: Windows 11 Enterprise Insider Preview(24H2)
OS Build: 26544.1000

### Repro Steps:

1. Launch application URL : https://makecode.microbit.org/
2. Home page will open.
3. Tab to "import" button and press enter key.
4.  Import dialog will open.
5. Tab to "Import file" Button.
6. Observe the issue.

**Note:** Similar issue is observed in below screens also:

1. Select language dialog 
2. History, Release Zone, Extension Zone Section in Review, Commit and push local changes to GitHub.
3. Sign in with GitHub dialog.

### Actual:
The keyboard focus indicator "Import file", "Import GitHub repo", "Import Url" button within Import dialog has a contrast ratio of (1.248:1) against its background, which is below the required minimum of 3:1.

### Expected result:
The keyboard focus indicator should have a minimum contrast ratio of 3:1 against adjacent background colors, ensuring it is clearly visible when navigating using the keyboard.

### User impact:
Keyboard-only users and users with low vision may not be able to identify the current focus location, leading to difficulty in navigation, confusion, and potential activation of unintended controls.

### Attachment:

<img width="826" height="417" alt="Image" src="https://github.com/user-attachments/assets/df2b31d3-7676-400c-bbd7-d0b74c11c400" />

<img width="830" height="416" alt="Image" src="https://github.com/user-attachments/assets/59b5d3d2-d3d0-4b4a-911d-3ffbfd99492e" />

<img width="820" height="416" alt="Image" src="https://github.com/user-attachments/assets/6fd402a6-25ed-4f66-a72a-81ad3f95c209" />

**Comment by riknoll** [comment]
fixed by pxt#11424
